sr_ethercat_hand_config/calibrations/rh/calibration.yaml

```diff
@@ -14,13 +14,39 @@
 ["LFJ1", [[3281.0, 0.0], [2607.0, 22.5], [1798.0, 45.0], [834.0, 67.5], [342.0, 90.0]]], 
 ["LFJ2", [[2793.0, 0.0], [2553.0, 22.5], [2202.0, 45.0], [1834.0, 67.5], [1462.0, 90.0]]], 
 ["LFJ3", [[1514.0, 0.0], [1985.0, 22.5], [2493.0, 45.0], [2956.0, 67.5], [3223.0, 90.0]]], 
-["LFJ4", [[2039.0, -20.0], [1941.0, -10.0], [1702.0, 0.0], [1249.0, 10.0], [626.0, 20.0]]], 
-["LFJ5", [[1491.0, 0.0], [1876.0, 22.5], [2238.0, 45.0], [2818.0, 67.5]]], 
-["THJ1", [[2678.0, 0.0], [2240.0, 22.5], [1760.0, 45.0], [1421.0, 67.5], [1162.0, 90.0]]], 
-["THJ2", [[1714.0, -40.0], [1807.0, -20.0], [1934.0, 0.0], [2074.0, 20.0], [2188.0, 40.0]]], 
+["LFJ4", [[2039.0, -20.0], [1941.0, -10.0], [1702.0, 0.0], [1249.0, 10.0], [626.0, 20.0]]],
+["LFJ5", [[1491.0, 0.0], [1876.0, 22.5], [2238.0, 45.0], [2818.0, 67.5]]],
 ["THJ3", [[1462.0, -15.0], [2145.0, 0.0], [2578.0, 15.0]]], 
 ["THJ4", [[1731.0, 0.0], [2047.0, 22.5], [2328.0, 45.0], [2571.0, 67.5]]], 
 ["THJ5", [[575.56, -60.0], [1196.295, -30.0], [1967.305, 0.0], [2701.42, 30.0], [3189.07, 60.0]]], 
 ["WRJ1", [[2542.0, -45.0], [2375.0, -22.5], [2139.0, 0.0], [1973.0, 15.0], [1814.0, 30.0]]], 
-["WRJ2", [[3083.0, -30.0], [1713.0, 0.0], [1418.0, 10.0]]], 
-]}
\ No newline at end of file
+["WRJ2", [[3083.0, -30.0], [1713.0, 0.0], [1418.0, 10.0]]]
+]
+
+sr_calibrations_coupled: [
+  [["THJ1", "THJ2"], [[[2738, 2301], 0.0, 39.99818],
```

Comment:
Please round up the numbers ;)
The calibration points are defined by us in degrees. See the numbers in the old calibration above.